services/metadactyl-clj/src/metadactyl/user.clj

```diff
@@ -17,9 +15,9 @@
   (log/debug user-attributes)
   (let [uid (user-attributes :user)]
     (if (empty? uid)
-      (throw+ {:error_code common-errors/ERR_NOT_AUTHORIZED,
-               :user (dissoc user-attributes :password),
-               :message "Invalid user credentials provided."}))
+      (throw+ {:type :clojure-commons.exception/bad-request-field
+               :error "Invalid user credentials provided."
+               :user (dissoc user-attributes :password)}))
```

Comment:
@ianmcorvidae You mean like the line immediately after this?
`(select-keys user-attributes [:password :email :first-name :last-name])`